docs/product/components/buttons.html

```diff
@@ -233,6 +233,92 @@
     </div>
 </section>
 
+<section class="stacks-section">
+    {% header "h2", "Vote" %}
+    <p class="stacks-copy">TODO describe</p>
+
+    <div class="stacks-preview">
+{% highlight html %}
+<button class="s-btn s-btn__vote" type="button">
```

Comment:
I think we should add the appropriate aria attributes to this example since it is used in most places as a toggle: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed